packages/actor-tests/bedrock/account-sends.ts

```diff
@@ -0,0 +1,34 @@
+import { Wallet, providers } from 'ethers'
+import { expect } from 'chai'
+
+import { actor, setupActor, run } from '../lib/convenience'
+import { devWalletsL2 } from './utils'
+
+interface Context {
+  wallet: Wallet
+}
+
+actor('Sender', () => {
+  let provider: providers.JsonRpcProvider
+
+  let wallets: Wallet[]
+
+  setupActor(async () => {
+    provider = new providers.JsonRpcProvider(process.env.L2_RPC)
+    wallets = devWalletsL2()
+  })
+
+  run(async (b, ctx: Context, logger) => {
+    const sender = wallets[Math.floor(Math.random() * wallets.length)]
```

Comment:
With enough concurrency, this could hit nonce issues. Is there a way to ensure that `wallets.length === concurrency` and then deterministically select a wallet?

Reply:
This is tough - in previous iterations of this script it would fund intermediate wallets to make sure that nonces don't collide. I can have it do that again here.